spec/helpers/recommendations_helper_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+describe RecommendationsHelper do
+  describe "recommendation_valid?" do
+    context "users and project exist" do
+      let!(:project_1) { FactoryGirl.create(:project) }
+      let!(:project_2) { FactoryGirl.create(:project) }
+
+      it "returns false when recommendation class and recommendation_for class
+        are the same" do
+        expect(recommendation_valid?(project_1, project_2)).to eq false
+        expect(recommendation_valid?(
+                project_1.user, project_2.user)).to eq false
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than `recommendation_valid?(`.